src/services/test-helpers/mock/accounts/eraRewardPoints.ts

```diff
@@ -0,0 +1,21 @@
+import { BTreeMap } from '@polkadot/types';
+
+import { polkadotRegistryV9110 } from '../../../../test-helpers/registries';
+
+const rewardOne = polkadotRegistryV9110.createType('u32', 3360);
+const rewardTwo = polkadotRegistryV9110.createType('u32', 4440);
+const mapToBe = new Map();
+mapToBe.set('12JZr1HgK8w6zsbBj6oAEVRkvisn8j3MrkXugqtvc4E8uwLo', rewardOne);
+mapToBe.set('1HDgY7vpDjafR5NM8dbwm1b3Rrs4zATuSCHHbe7YgpKUKFw', rewardTwo);
+const bTree = new BTreeMap(
+	polkadotRegistryV9110,
+	'AccountId32',
+	'u32',
+	mapToBe
+);
```

Comment:
How does the `BTreeMap` type work? Do these 4 args become two key/value pairs or…?

Reply:
So here the `BTreeMap`, is taking in a `Map` object and will convert it's key, values into the BTree. This is held in the `mapToBe` variable

Reply:
Probably better being `mappedValues` or something

Reply:
Switched the naming to `mockEraRewardsMap`